### ISSUE TITLE
Make authors of the blog post the link to their info

### ISSUE THREAD

**Issue by vrom911** [kowainik/kowainik.github.io#27]
Each author name should be a link to the main page `/team/#name.surname`

**Comment by sphaso** [comment]
I see two issues with this:
- the team member modal is currently just that, so we'd need some javascript to act as a router and open the modal
- the haskell / template part to generate the link :)

Do you agree? Is there any other way you thought to approach this?

**Comment by vrom911** [comment]
@sphaso Yes, I see now that it's a bit complicated as should involve some JS code.  For the link part, I was thinking that it's possible to reuse `Team` module we have as it already has all this information. The thing now a bit more difficult as I thought it would be, cause we have some general posts which are authored as Kowainik, also be can have blog posts with several authors.
Do you think these complications worth it, or that it's doable without tears?

**Comment by sphaso** [comment]
@vrom911 hopefully no tears :)
1. Regarding general posts, I guess it's just a conditional on the link creation. One could point to the `#team` route
2. To do this we might consider adopting a JS router for reduced tears as coding one by hand is not trivial
